src/main/java/pl/put/poznan/transformer/logic/text/abbreviations/Inverse.java

```diff
@@ -0,0 +1,53 @@
+package pl.put.poznan.transformer.logic.text.abbreviations;
+
+
+import pl.put.poznan.transformer.logic.text.Text;
+import pl.put.poznan.transformer.logic.text.TextTransformer;
+import java.lang.*;
+
+
+/**
+ * Inverse text
+ */
+public class Inverse extends TextTransformer {
+    public Inverse(Text text){
+        super(text);
+    }
+
+    @Override
+    public String transform(){
+        return reverse(text.transform());
+    }
+
+    /**
+     * Reverse given text
+     *
+     * @param text holds the text
+     * @return reversed text
+     */
+    private String reverse(String text){
+        char[] temparray = text.toCharArray();
```

Comment:
Tu by się przydała bardziej verbose nazwa zmiennej. Co trzyma ta zmienna? Co się w niej znajduje i co będzie się znajdować?

PS. w javie notacja to raczej `tempArray` :wink: